### PR TITLE
Revamp Cloud Quickstart + CLI Quickstart

### PR DESCRIPTION
**Pull request by jwitz**
Changes specific to the CLI guide.

### CODE REVIEW

File: cloud/stable/01_get-started/01_quickstart.md [hunk 1]
```diff
@@ -316,24 +259,78 @@ $ astro deploy
...
-Once you deploy to Astronomer, navigate back to the [Astronomer UI](https://app.gcp0001.us-east4.astronomer.io/). From there, go to **Deployment** > **Settings** > **Open Airflow**.
+Once you deploy to Astronomer, reopen the [Astronomer UI](https://app.gcp0001.us-east4.astronomer.io/). From there, go to **Deployment** > **Settings** > **Open Airflow**.
 
 ![Access Airflow UI](https://assets2.astronomer.io/main/docs/getting-started/open-airflow.png)
 
-This links to your Deployment's Airflow UI, where you'll find your `example_dag` and all other DAGs in your project.
+This links to your Deployment's Airflow UI, where you'll find your example DAG and all other DAGs in your project.
 
-For guidance on how to navigate the Airflow UI, refer to [Apache Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/ui.html).
+For guidance on how to navigate the Airflow UI, refer to the [Apache Airflow documentation](https://airflow.apache.org/docs/apache-airflow/stable/ui.html).
```
Comment (paolaperaza) [inline cloud/stable/01_get-started/01_quickstart.md:270]: No "the"?

File: cloud/stable/01_get-started/01_quickstart.md [hunk 2]
```diff
@@ -316,24 +259,78 @@ $ astro deploy
...
+
+In addition to customizing the Astronomer Certified image referenced in your Dockerfile, you can:
+
+- Add DAGs to the `dags` directory
+- Add custom Airflow plugins to `plugins`
+- Add Python packages to `requirements.txt`
+- Add OS-level packages to `packages.txt`
+- Add Environment Variables to your `Dockerfile` ([guidelines](https://forum.astronomer.io/t/how-do-i-set-astronomer-config-file-options-env-vars/186/2))
+
+### Review the Astronomer Certified System Distribution
```
Comment (paolaperaza) [inline cloud/stable/01_get-started/01_quickstart.md:315]:
```suggestion
### Customize your Astronomer Certified Image
```

File: cloud/stable/01_get-started/01_quickstart.md [hunk 3]
```diff
@@ -316,24 +259,78 @@ $ astro deploy
...
+Astronomer Certified supports both [Alpine Linux](https://alpinelinux.org/) and [Debian](https://www.debian.org/)-based images for versions 1.10.5-1.10.12. In an effort to standardize our offering and optimize for reliability, we'll exclusively build, test and support Debian-based images starting with AC 1.10.14.
+
+If you're unfamiliar with Alpine Linux or Debian, check out some examples of what you might need based on your use-case:
+
+- [GCP](https://github.com/astronomer/airflow-guides/tree/main/example_code/gcp/example_code)
+- [Snowflake](https://github.com/astronomer/airflow-guides/tree/main/example_code/snowflake/example_code)
+
+To learn more, refer to [Manage Airflow Versions](/docs/cloud/stable/customize-airflow/manage-airflow-versions/).
+
+### Complete Additional Setup
```
Comment (paolaperaza) [inline cloud/stable/01_get-started/01_quickstart.md:326]: Don't love this title but can't think of anything better - will ponder 🤔

**Review by paolaperaza — CHANGES_REQUESTED** [review]
Thanks @jwitz ! Made a few comments. I didn't get a chance to review changes for the Astro CLI doc - can you put that in another PR to keep things simple?

File: cloud/stable/01_get-started/01_quickstart.md [hunk 4]
```diff
@@ -8,57 +8,53 @@ Welcome to Astronomer.
...
+For more information on Workspace Admins, Deployment Admins, and other user roles and permissions, refer to the [Manage User Permissions on Astronomer Cloud](/docs/cloud/stable/manage-astronomer/workspace-permissions/) guide.
 
-## Start with the Astronomer CLI
+## 4. Install the Astronomer CLI
 
 The [Astronomer CLI](https://github.com/astronomer/astro-cli) is the easiest way to run Apache Airflow on your machine. From the CLI, you can establish a local testing environment and deploy to Astronomer Cloud whenever you're ready.
 
-### Install
 
-#### Prerequisites
```
[the remark this answers is not in the document]
Comment (jwitz) [reply cloud/stable/01_get-started/01_quickstart.md:57]: I split the two different installations (Homebrew and cURL) into separate headers, which I think breaks things up and makes more sense!

File: cloud/stable/01_get-started/01_quickstart.md [hunk 5]
```diff
@@ -8,57 +8,53 @@ Welcome to Astronomer.
...
-If you're the first person on your team to sign up for Astronomer, create a Workspace to officially begin your 14-day trial.
+If you're the first person on your team to sign up for Astronomer, click **New Workspace** to create a Workspace and officially begin your 14-day trial.
 
-![Create an Workspace on Astronomer](https://assets2.astronomer.io/main/docs/getting-started/create-workspace.png)
+![Create a Workspace on Astronomer](https://assets2.astronomer.io/main/docs/getting-started/create-workspace.png)
 
 You can think of Workspaces the same way you'd think of teams - a space that specific user groups have access to with varying levels of permissions. From within a Workspace you can create one or more Airflow Deployments, each of which hosts a collection of DAGs.
 
-For more information, refer to [Manage Workspaces and Deployments](/docs/cloud/stable/deploy/manage-workspaces/).
+For more information, refer to the [Manage Workspaces and Deployments](/docs/cloud/stable/deploy/manage-workspaces/) guide.
```
[the remark this answers is not in the document]
Comment (jwitz) [reply cloud/stable/01_get-started/01_quickstart.md]: Deferring to the google style guide on this one: simply referring to the title of the guide without a noun after it https://developers.google.com/style/cross-references?hl=en

File: cloud/stable/01_get-started/01_quickstart.md [hunk 6]
```diff
@@ -8,57 +8,53 @@ Welcome to Astronomer.
...
-This is how you'll log into both the Astronomer UI and the CLI in the future. 
+This is how you'll log into both the Astronomer UI and the CLI in the future.
 
 > **Note:** Once you've created an account on Astronomer Cloud, you will NOT be able to change your method of authentication.
 
-### Create a Workspace
+## 3. Create a Workspace
 
-If you're the first person on your team to sign up for Astronomer, create a Workspace to officially begin your 14-day trial.
+If you're the first person on your team to sign up for Astronomer, click **New Workspace** to create a Workspace and officially begin your 14-day trial.
```
[the remark this answers is not in the document]
Comment (jwitz) [reply cloud/stable/01_get-started/01_quickstart.md:35]: I always consider "Select" to involve a choice, as in "select from the following options". In this context, click is the most accurate word to use.

File: cloud/stable/01_get-started/01_quickstart.md [hunk 7]
```diff
@@ -8,57 +8,53 @@ Welcome to Astronomer.
...
-![Create an Workspace on Astronomer](https://assets2.astronomer.io/main/docs/getting-started/create-workspace.png)
+![Create a Workspace on Astronomer](https://assets2.astronomer.io/main/docs/getting-started/create-workspace.png)
 
 You can think of Workspaces the same way you'd think of teams - a space that specific user groups have access to with varying levels of permissions. From within a Workspace you can create one or more Airflow Deployments, each of which hosts a collection of DAGs.
 
-For more information, refer to [Manage Workspaces and Deployments](/docs/cloud/stable/deploy/manage-workspaces/).
+For more information, refer to the [Manage Workspaces and Deployments](/docs/cloud/stable/deploy/manage-workspaces/) guide.
 
-#### Join a Workspace
+#### Join an Existing Workspace
```
[the remark this answers is not in the document]
Comment (jwitz) [reply cloud/stable/01_get-started/01_quickstart.md]: Fixed in my latest commit!

File: cloud/stable/01_get-started/01_quickstart.md [hunk 8]
```diff
@@ -116,21 +118,19 @@ The Docker image you'll find by default in your Dockerfile is:
...
 
-To help you get started, your initialized project will come with an "Example DAG" in `/dags` by default. The DAG itself simply prints today's date, but it'll give you a chance to get accustomed to our deployment flow.
-
-If you'd like to deploy some more functional example DAGs, upload your own or check out [example DAGs we've open sourced](https://github.com/airflow-plugins/example-dags).
-
-## Develop Locally
+To help you get started, your initialized project includes an example DAG in `/dags`. This DAG simply prints today's date, but it'll give you a chance to become familiar to deploying on Astronomer.
 
-With those files in place, you're ready to push your Airflow image to your local Airflow environment.
+If you'd like to deploy some more functional DAGs, upload your own or check out [example DAGs we've made open source](https://github.com/airflow-plugins/example-dags).
```
[the remark this answers is not in the document]
Comment (jwitz) [reply cloud/stable/01_get-started/01_quickstart.md]: I think it's fine either way, but using it as a verb is slightly more informal. If we're cool with that, I'll keep it in.

**Review by paolaperaza — APPROVED** [review]
Alrighty, @jwitz ! Mergin this.

**Pull request merged**